.circleci/config.yml

```diff
@@ -39,11 +41,19 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Verify that requirements.txt contains the right dependencies for this python version
+          name: Verify that requirements.txt contains the right dependencies
+            for this python version
           command: |
             pip install pip-tools
             pip-compile --quiet --generate-hashes requirements.in
             git diff --exit-code requirements.txt
+  yamllint-test:
+    docker: *docker 
+    steps: 
+      - checkout
+      - run: 
+          name: Verify that yaml files are linted for errors
+          command: PATH="venv/bin:$PATH" yamllint -c .yamllint.yaml
```

Comment:
Doing this is more complicated than before, because the environment doesn't have the requirements set up as a separate stage. It also has to be added to the workflows. I'd suggest reverting back to putting it before the pytest and linters.

Reply:
ok will do!